cips/cip-template.md

```diff
@@ -42,6 +42,18 @@ It is recommended to follow RFC 2119 and RFC 8170. Do not remove the key word de
 
 The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "NOT RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be interpreted as described in RFC 2119 and RFC 8174.
 
+## Parameters
+
+The parameters section should summarize any changes to global or module parameters, including any new parameters, introduced by the CIP. All mainnet parameters are tracked in [CIP-13](./cip-13.md). Once a CIP is accepted and deployed to mainnet, CIP-13 MUST be updated with these parameter changes. If there are no parameter changes in the CIP, this section can be omitted. 
+
+**TODO: Remove the previous comments and update the following table before submitting**
+
+| Parameter     | Default | Summary                                                                                                                | Changeable via Governance |
```

Comment:
I think there are two flavors of parameter changes:
1. changes to existing parameters
2. new parameters

For changes to existing parameters, I think this columns would be helpful:

```
| Parameter | Current value | Proposed value | Description | Changeable via Governance |
```

For new parameters the Current value column can be omitted:

```
| Parameter  | Proposed value | Description | Changeable via Governance |
```

I think the "Default" column is a bit confusing because the Default value for a parameter could be changed via a CIP

Reply:
Yup good point, will fix this. The column is weirdly called Default in CIP13 so maybe we should just change it to Value there